packages/react-reconciler/src/ReactFiber.js

```diff
@@ -4,7 +4,6 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
```

Comment:
It's my fault for missing this in the review, but this completely removed flow typing from the most important module in the codebase.

Reply:
@gaearon : Can I send a PR for adding flow typing here again?

Reply:
@gaearon for reference there's an existing PR https://github.com/facebook/react/pull/12764